.ci/azure-pipelines/build-macos.yml

```diff
@@ -44,12 +45,6 @@ jobs:
         displayName: 'CMake Configuration'
       - script: |
           cd $BUILD_DIR
-          # Compiling some of the test targets with -j2 option leads to pipeline failures
-          # (presumably out of memory error). Thus we make them separately in a single
-          # thread mode. Their corresponding modules are built before with the -j2 mode
-          # to make the process faster.
-          cmake --build . -- -j2 pcl_filters pcl_registration
-          cmake --build . -- test_filters test_registration test_registration_api
           cmake --build . -- -j2
```

Comment:
Did you try 3 or more jobs? With Release, lesser RAM is used, so maybe we can have more parallel jobs (same applies to other CI)

Reply:
No didn't try as the pipelines run [Standard_DS2_v2](https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series#dsv2-series) which have 2 vCPUs only, so adding more jobs beyond 2 shouldn't give any speedup.
Will benchmark though, just to be sure.

Reply:
I was actually looking for this info. Where did you find this?

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops

Reply:
Did some benchmarks, no speedup when jobs are increased beyond vCPU count.